kit/foundation/docs/template/docs/foundation/README.md

```diff
@@ -1,3 +1,88 @@
-# My Foundation
+# Welcome to Cloud Foundation
+
+## Your way to the cloud
+
+Cloud adoption has two phases:
+- The planning phase often begins with our consulting hours: Here you can inform yourself and ask your questions. In this phase you will choose a cloud provider and landing zone. Please note that not every service is available from every provider and landing zone. We recommend to create a solution design at this stage. Our cloud solution architects will support you in this.
+- The implementation phase begins when you have access to your cloud tenant. Now it's time for the big move - migrate your application and if necessary your data!
+
+## Cloud Providers
+
+> 🚧 Adapt the list of cloud providers you want to offer.
+
+Our strategic cloud partners are
+- AWS
+- Google
+- Microsoft
+
+We offer [landing zones](concepts.md#landing-zone) for each of those [platforms](/platforms/).
+
+## Shared Responsibility Model
+
+For our cloud offering, we rely on sharing responsibilities across three parties: cloud provider, cloud foundation team, and you (the application team).
+It’s critical to understand which security tasks are handled by whom.
+
+## Common responsibilities across all landing zones
+
+The following responsiblities are with the cloud provider for all landing zones:
+- Physical hosts
+- Physical network
+- Physical datacenter
+
+The following responsibilities are with the cloud foundation team for all landing zones:
+- Identities, SSO on cloud platform
+- Audit Logs for calls against cloud platform APIs
+
+The following responsibilities are with the applicaiton team for all landing zones:
+- Permissions on the cloud platform
+
+## Landing Zones and Responsibilities
+
+> 🚧 Adapt the list of landing zones you want to offer.
```

Comment:
Is there a resource we can link to for more details? The ultimate landing zone guide comes to mind, LZ101 or the CFMM.

Reply:
Add sentence: "In this example here we've chosen to divide landing zones by use case (Lift and Shift and cloud-native workloads) as well as network connectivity to on-prem.

Reply:
can also have an additional addendum: this division is a very common starting point for cloud foundation teams